internal/adapters/uidriver/socket_test.go

```
@@ -0,0 +1,150 @@
+	directory := t.TempDir()
+	require.NoError(t, os.Chmod(directory, 0o700))
+	path := filepath.Join(directory, "ui.sock")
+	server := New(service, nil)
+	endpoint, err := ListenUnix(path, server, func() Ready {
+		return testReady(false)
+	})
+	require.NoError(t, err)
+	require.Equal(t, os.FileMode(0o600), fileMode(t, path))
+	defer endpoint.Close()
+
+	conn, err := net.DialTimeout("unix", path, time.Second)
```

golangci-lint (2.13.2)

[error] net.DialTimeout must not be called. use (*net.Dialer).DialContext with (*net.Dialer).Timeout

(noctx)